mobile/src/modules/price-api/store/reducer.ts

```diff
@@ -52,6 +63,19 @@ export interface HistoricalPriceInfo {
   HasWarning?: boolean;
 }
 
+export enum PriceType {
```

Comment:
Nah, use
```
BURST = 'BURST',
BTC = 'BTC',
```

and in other places PriceType.BTC instead of plain strings like 'BTC'